kv_store/include/server_utility.h

```diff
@@ -33,22 +33,21 @@ struct pair_hash {
 // we have this template because we use responsible for with different hash
 // functions depending on whether we are checking local or remote
 // responsibility
+// New comment by chenggang: simplified the code
+// assuming the replication factor will never be greater than the number of nodes in a tier
```

Comment:
This seems reasonable, but instead of hardcoding these into the file (which requires recompilation to change), can we create a YAML config file with all the relevant paths and config params?